### DISCUSSION TITLE
Burn Baby Burn

### DISCUSSION THREAD

**Discussion by heytx** [handshake-org/HIPs#69] [Polls]
Based on community feedback, an alternative proposal is to be presented: **_Burn Baby Burn_**. There are currently 628M HNS in circulation, and 890M HNS left unclaimed for open source developers. The primary component is officially ending the open source developer airdrop. The secondary component is making ICANN TLDs and Alexa Top 1K claimable, which requires a hardfork. Overall, this proposal is aligned with the Handshake ethos due to the absence of a centralized foundation or group receiving an airdrop reallocation.

### Burning Open Source Developer Airdrop
Rather than reallocating the remaining 890M HNS airdrop, this will be officially removed from the total supply. The arguments for ending the open source developer airdrop include:
- This airdrop has been publicly available for more than four years;
- Only 2.9% of the airdrop was claimed, most of which occurred during the first year of Handshake’s launch;
- This provides certainty of coin issuance, which will increase confidence among miners and $HNS holders;
- This is not an effective mechanism for developer adoption.

Having coin issuance solely derived from POW miners can strengthen $HNS price stability. Removing the possibility of $HNS being created at a zero-cost basis, whether from the open source developer airdrop or name claims, will effectively establish Handshake as an authentic POW chain once-and-for-all. $HNS trending to zero over the past few years is arguably based on the murmurings of this free coin issuance and shroud of the actual coin supply. Rather than allowing these original traits determine the collective price of circulating $HNS, _**burning**_ will free $HNS from the shackles. The value of $HNS will more accurately be based on the utility of the coin (eg. bidding, renewals, updating records), TLDs in existence, and overall adoption of the protocol. By $HNS increasing in price, this should result in more developers and users overall. 

### Remaining Name Claims
With the soft fork of February 2024, the unclaimed Alexa Top 10K and ICANN TLDs were locked. This hard fork proposal includes unlocking the Alexa Top 1K and ICANN TLDs for claiming. ICANN TLDs will be permanently reserved for the ICANN TLD owners, and the Alexa Top 1K will be reserved for the ICANN domain counterpart (eg. openai.com) for an additional four years from the hard fork date. While initial $HNS allocations were included for claiming reserved names, this early adoption mechanism will be removed for these remaining names. In order to update records, transfers, or renewals, $HNS will need to be acquired on exchanges at market prices. The previously allocated 188.3M $HNS, while technically never in existence, will effectively be deducted from the total supply. The Alexa 1,001-10,000 will become available for auction at the commencement of the hard fork.

### Adjusted HNS Coin Total Supply
Based on August 25, 2024 (blockheight 239,664) [1]:
- Circulating supply: 628M HNS
  - issued: 690.9M HNS
  - previously burned: 62.9M HNS
- **Burn Baby Burn Proposal** 
  - Burned airdrop: 890.4M HNS
  - Removed unclaimed name claims: 188.3 M HNS
- Unmined block rewards: 270.3M HNS
- Previous Total Supply: 2,040M HNS
- **New Total Supply: 898.3M HNS**

### Fee and Renewal Limit Adjustments
This proposal also includes lowering the renewal limit from 600 to 200 per block. Instead of renewals being .1 $HNS every two years, this will effectively create a competitive fee market for miners and TLD owners. This renewal limit adjustment will have a two-fold effect by increasing demand for $HNS and hashrate to secure the network.

### ReRecord On-chain (HIP-018)
A and AAAA records can be added to the TLD on-chain. This “allows users to point to an IP address (host), on the HNS blockchain, without needing to also setup or use a name server service, at the bare TLD level” [2] [3]. Here are the benefits for allowing A AAAA records on-chain:
- More choice to TLD owners 
- Trustless in not having to rely on 3rd party nameservers or services
- More utility to HNS blockchain
- Increases blockchain usage, miner fees, and $HNS demand

### Implementation
If there is community consensus, including support from Namebase, @nodech has agreed to write the code and be the lead developer. There should also be additional developers to review and assist where needed. This support may come from existing members or outside the community. Hey TX is willing to contribute towards implementation efforts. The community can raise the remaining funds necessary to cover costs as needed. Per @pinheadmz, funding for core developers should derive from outside the chain.

As a reference, below is a job description for a core developer: 
Cryptocurrency Protocol Engineer: Add features, test, and maintain a layer-1 blockchain daemon based on Bitcoin (UTXO/PoW) written in NodeJS.

[1]  https://hns.cymon.de/stats
[2]  https://rerecord.skyinclude.com
[3] https://github.com/skyinclude/HIPs/blob/HIP018-Adding-A-AAAA-Records/HIP-XXXX-adding-A-AAAA-records.md

**Comment by heytx** [comment]
Importantly, the one component of this proposal that requires a hard fork is making the ICANN TLDs and Alexa Top 1K claimable again. I view this as necessary to do at some point, which is why it’s included. The community can decide through discussion on whether to do a soft or hard fork though.

Generating excitement through increased TLD utility, the Alexa 1-10K available for auction, and effectively reducing the total $HNS supply by 56% can result in growing our community and funding development. 

In addition to being prepared to cover some developer costs in the implementation of this proposal, there should also be a community fund to pay news publications to cover the protocol changes in order to attract new users and builders. There can be HS3 miner giveaways to developers and the overall community.

**Reply by DIPMR** [reply]
Add one more thing if you can - Release names over a period of time not like what happened earlier this year.

**Reply by AndreiChyrych** [reply]
What’s your thoughts about adding support for MX records as well, @heytx?

**Reply by heytx** [reply]
I'm not against adding more records (eg. mx), although I would want to hear counter viewpoints on whether this could potentially bloat the chain too much.

**Reply by AndreiChyrych** [reply]
It looks like adding additional on chain records might drive demand for block space which might result in higher demand for the HNS coin. 

What do you mean with bloating the chain, @heytx ?

**Reply by Falci** [reply]
The more data we store on chain, the more disk space is needed to run a full node. There is a point where it becomes too expensive to run a full node. Not saying we would reach this point if we add mx records but it's something to keep in mind

**Reply by AndreiChyrych** [reply]
Every TLD can be 512 bytes already now, right? So I thought that adding different types of records will not change other properties significantly since you can’t have more than 512 bytes per TLD one way or another.

**Reply by Falci** [reply]
Yes, they can have up to 512 bytes. But even without changing the limit we would be incentivising its usage.
There's no magic, this data needs to be stored somewhere.

**Reply by AndreiChyrych** [reply]
You think that 512 bytes per TLD is too much long term, @Falci ?

**Reply by eskimo** [reply]
If millions of TLDs actually get used, yeah, the amount of storage needed will be insane.

**Reply by AndreiChyrych** [reply]
What do you think about dynamic fees, @eskimo ? The more records you use, the higher the fee is. 

Or what about simply decreasing TLDs weight from 512 to something lower?

**Reply by eskimo** [reply]
I said this in the discord yesterday to have UPDATE and REWEW costs be based on the amount of storage being used.

**Reply by pinheadmz** [reply]
> I said this in the discord yesterday to have UPDATE and REWEW costs be based on the amount of storage being used.

They already are, by transaction fee rate which is total fee divided by size of the transaction.

**Reply by eskimo** [reply]
> > I said this in the discord yesterday to have UPDATE and REWEW costs be based on the amount of storage being used.
> 
> They already are, by transaction fee rate which is total fee divided by size of the transaction.

Yeah I mean like even more though

**Reply by pinheadmz** [reply]
I had a crazy idea that a rule could be added that requires urkel proofs to be included somewhere in renew transactions. this would mean that as more names get added to the urkel tree, all renewal transactions would get bigger and therefore more expensive

**Comment by pcfreak30** [comment]
I am definitely against this proposal. burning is virtue signaling and a real waste of energy/time and only helps traders in the short term.

HNS needs funding, not 1 day of PR for an article saying we burn X million coins, which won't help anything.

**Reply by heytx** [reply]
Burning can potentially have a positive long-term effect, as what's highlighted in the proposal. Let's circle back to recent conversations: the only consensus reached for a reallocation was to fund core developers. However, the actual core devs for HNS are against this! Zipkin's view is that funding must come from outside the chain. This proposal creates an environment where this is possible.

**Reply by eskimo** [reply]
Burning has never once caused long term effects, if it did, HNS would be at $100 given that 70m HNS has been burned.

**Reply by heytx** [reply]
Fair enough, the burning mechanism via auctions hasn't had a desirable effect so far. This is 10x greater though and would remove sell pressure from those that obtain the coin at a zero-cost basis. For example, Brave claimed 3,400,503.51 $HNS back in 2020, and dumped it on the market for more than half a million dollars. That will no longer be possible, which can provide reassurance to miners and investors.

**Reply by eskimo** [reply]
The price going up at all will create an equal amount of sell pressure, which is the main reason the price is what it is. Nobody is buying, everybody wants an exit.

**Reply by pcfreak30** [reply]
@heytx   ```Burning can potentially have a positive long-term effect, as what's highlighted in the proposal. Let's circle back to recent conversations: the only consensus reached for a reallocation was to fund core developers. However, the actual core devs for HNS are against this! Zipkin's view is that funding must come from outside the chain. This proposal creates an environment where this is possible.```

I get you are being diplomatic. Zipkin I will assume has a more BTC purity view than pragmatic stance. Regardless, im against and its a waste of time and doesn't actually solve anything. I would rather do nothing then do a burn if those were my options.

**Reply by TexasShaker** [reply]
No one will buy an HNS ASIC miner unless HNS is scarce.  Scarcity is what makes BTC valuable.  Scarcity is the foundation of free market economics.  Burn as much as possible.  And give away ASICs like they did muskets in 1776.  I also am in favor of decentralizing the SLDs on Handshake.  We can't rely on other blockchains to mint SLDs.  Nole has my full support.

**Reply by mtomich** [reply]
> The price going up at all will create an equal amount of sell pressure, which is the main reason the price is what it is. Nobody is buying, everybody wants an exit.

Yep. There is absolutely, postively, zero value to burning tokens. This isn't a meme token it is a utility token. The project succeeds by creating token value, not by destroying the supply.

**Comment by eskimo** [comment]
Burning is idiotic. It's a no from me.

**Reply by heytx** [reply]
Okay, there is strong sentiment within the community for burning. We should consider this viewpoint.

**Reply by eskimo** [reply]
The only reasons people want to burn are greed or jealousy. Greed because they think it'll raise the price (even though it won't) and jealously because they won't coins if there's a reallocation. 99% of people here are either greedy or jealous so that explains why they're for it.

**Reply by heytx** [reply]
The majority of the community won't ever support a reallocation; the core devs don't want any form of a multisig or reallocation. A lot of that comes down to trust and centralization, which is understandable. At some point, we must listen to the community. Continuing with pure decentralization at the protocol level is likely the best long-term approach.

**Reply by eskimo** [reply]
Ok well burning the coins will cause a tiny spike in price which will cause most to sell and then essentially making the market cap a fraction of what it currently is after the price crashes down even further. The community doesn't matter, the only people that should have a vote are people that have built a significant amount of stuff or have put in a lot of money. Someone who is holding 10k garbage names or owning $15 of HNS has a worthless opinion tbh.

**Reply by ForestCoin** [reply]
@eskimo, your contributions to the Handshake community have been remarkable! You've developed HNSChat, ShakeStation, and other tools, and I frequently use ShakeStation — it’s very well done! However, after four years of development, Handshake's user base has been shrinking, as you might already know. It's time for us to make a change! I believe Handshake represents the future, and TLDs are the true Web3!

**Reply by eskimo** [reply]
> Handshake's user base has been shrinking, as you might already know. It's time for us to make a change!

I agree, but burning will do absolutely nothing but sink the sinking ship even faster.

**Reply by pcfreak30** [reply]
@ForestCoin I appreciate you having a positive view, but burning appeases only 1 group, hodler's/traders and does zero for the R&D of the chain or anything else that helps HNS other that an hour of hype.

So im hard against any burn or any spin of why we should have a burn.

**Reply by eskimo** [reply]
> burning appeases only 1 group, hodler's/traders

Only temporarily too, long term the ship still sinks.

**Reply by marioo-lab** [reply]
> The only reasons people want to burn are greed or jealousy. Greed because they think it'll raise the price (even though it won't) and jealously because they won't coins if there's a reallocation. 99% of people here are either greedy or jealous so that explains why they're for it.

The burn could help everyone take their eyes of the coins, and focus on adoption. And I wouldn't talk about greed if I'm one of the targets of the so called "fund". Handshake is a decentralized protocol, and in order to remain so, no governing authority should be created, yet alone give it 50% of supply. At this point no one who wants this is thinking straight and is blinded by greed.

**Reply by ForestCoin** [reply]
@pcfreak30 Holders and traders are also part of the community! ETH staking requires holders to participate, and MakerDAO's voting needs token holders to get involved! If there are no holders promoting Bitcoin, who would know about its existence in the world? The more people hold it, the better it is for the project's development. 
We hope for the widespread adoption of TLDs, but if nothing is done, the number of users will inevitably decrease. Therefore, I support burn

**Reply by pcfreak30** [reply]
`And I wouldn't talk about greed if I'm one of the targets of the so called "fund"` IIRC eskimo got added to proposals without him being asked. Hell heytx basically asked me if Lume would want money, in these forums, and I said no as well.

What it does come down to, is some bag holders making noise to cash out.

And there does need to be some sort of governance/authority if only to accurately coordinate any significant funding. Taking a BTC maxi view right now is just not realistic to solve the problems HNS has. If HNS does not have any protocol level funding, or wont get it, then any centralized entities are not needed, as they are just a glue.

And the real issue is everyone wants HNS to grow, but no one really has the funding to hire, well.. anyone.

So one group wants NgU (many here responding :fire: :rocket: ) , one group wants a L2 and to give money away, and another generally just wants to fix the fact that the project has no actual funding.

**Reply by pcfreak30** [reply]
> @pcfreak30 Holders and traders are also part of the community! ETH staking requires holders to participate, and MakerDAO's voting needs token holders to get involved! If there are no holders promoting Bitcoin, who would know about its existence in the world? The more people hold it, the better it is for the project's development. We hope for the widespread adoption of TLDs, but if nothing is done, the number of users will inevitably decrease. Therefore, I support burn

Comparing to ETH is fairly different now since most there just use it to get yield solely as a passive investment, while delegating to ETH's version of miners to do the hard work of managing infra. So not comparable and to me doesn't count as someone being in the community unless they actually contribute outside of holding some coins.

So to me, anyone who is a HNS holder but isn't contributing besides being a bag holder isn't a voice I can justify to listen to, especially when they only come out of the wood work when something in the tokenomics to be changed is proposed...

**Reply by marioo-lab** [reply]
> ` IIRC eskimo got added to proposals without him being asked. Hell heytx basically asked me if Lume would want money, in these forums, and I said no as well.`
I didn't say he added himself, lol, but if you are part of the target group, how can you cheer for the 50% allocation and blame greed at the same time? Isn't this a bit hypocritical?

> `What it does come down to, is some bag holders making noise to cash out.`
Some may, as some are cheering to get some of that allocation. But in the end the burn doesn't airdrop money to anyone.

> `And there does need to be some sort of governance/authority if only to accurately coordinate any significant funding.` 
An authority controlling 50% of supply is one of the BIGGEST problems. Why not burn 90% of it and govern only 5-10% for "development" purposes, then? Surely 5% should be enough if no recipient wants to sell, and everyone believes it will have a great future afterwards.

**Reply by eskimo** [reply]
> I didn't say he added himself, lol, but if you are part of the target group, how can you cheer for the 50% allocation and blame greed at the same time? Isn't this a bit hypocritical?

I talked even more shit about that proposal than this one, and yeah, I didn't request to be added and didn't know I was even on it until after someone pointed it out to me. I wouldn't have accepted it regardless.

**Reply by pcfreak30** [reply]
`I didn't say he added himself, lol, but if you are part of the target group, how can you cheer for the 50% allocation and blame greed at the same time? Isn't this a bit hypocritical?` i dont think eskimo ever actually was for the L2 proposal b/c hes anti-L2... So he what?

`Some may, as some are cheering to get some of that allocation. But in the end the burn doesn't airdrop money to anyone.` That might be true, though its also unknown who was added to that L2 giveaway list without asking. And the burn doesnt do anything for HNS at all. So I do stand in the position of do nothing if the burn is the only other route, as the real issues are funding, not who should get money in the community, and not if a burn will help HNS (it wont).

`An authority controlling 50% of supply is one of the BIGGEST problems. Why not burn 90% of it and govern only 5-10% for "development" purposes, then? Surely 5% should be enough if no recipient wants to sell, and everyone believes it will have a great future afterwards.`

Or just reallocate 5-10%. I really don't care. I don't want to throw money away for no reason other than a CT shill post and a post on cointelegraph, and then still not solve the problem that we need both protocol and ecosystem dev $.

For a reference point, Sia, which has a 501c3 foundation, gets roughly 5 mil a year in block rewards, is spending 1-2 mil a year, and has a huge onchain multisig. And they are making software that's actually works and will be able to get adoption. I am grant funded by them, I will readily admit that, but I see a model as a reference point that is working and is rebooting itself.

Frankly the only difference _between_ Sia and HNS, is HNS is broke.

**Comment by heytx** [comment]
There are a lot of community suggestions to allow for all records on-chain, including MX and ALIAS. As we need to increase users, this can make Handshake more user-friendly and desirable. For those against this, provide your counterpoints. Otherwise, the proposal will be treated as allowing for these other records.

**Reply by evbots** [reply]
can we just let the record type be completely arbitrary? meaning the "type" can just be a string of less than N characters. This will allow TLDs to be used for any creative purpose.

**Reply by pcfreak30** [reply]
I personally don't see an issue with that unless you are trying to stay DNS complaint. Then again, DNS providers are inventing record types as-is. Im personally not that concerned with the disk space, compared to everything else I see in the industry abusing ledgers.

**Reply by eskimo** [reply]
Yeah I was going to say something negative about this but then thought about how I use custom record types on varo lol. I don't think it would be an issue, but also, we only have 512 bytes per TLD, so stuff on chain should be important stuff, ~30+ records or so will still need you to run a NS or use varo.

**Comment by ForestCoin** [comment]
I fully support this proposal! Handshake needs to burn tokens to restore community confidence! When I bought ETH in 2015, it was priced at $0.7. At that time, ETH didn't have many developers, and no development teams were interested in joining. But when ETH reached $1,000, it became the Layer 1 blockchain with the most developers in the industry! That's how this industry works: price increases attract a majority of speculators, but also attract some value investors who start researching where the project’s value lies. Some people will stay and develop… Handshake has been around for so many years without any influence in the industry; it's time to make a change! I fully support you.

**Reply by pcfreak30** [reply]
`Handshake needs to burn tokens to restore community confidence!` LOL, no. Confidence is increased when things are getting made that actually create adoption (tech, not investors). NgU should not be the primary approach for that.

**Reply by ForestCoin** [reply]
Handshake is crucial for the development of Web3, but the slow pace of technical advancements over the past four years has led to very few people using the Handshake network. The on-chain transaction volume speaks for itself. 
You can check the network data of Handshake;  data: https://e.hnsfans.com/charts
 if no changes are made, it will be difficult for miners to sustain operations, and the on-chain transaction volume will continue to decrease. Burning coins can boost community confidence, attract more miners, and make the network more secure.

**Reply by eskimo** [reply]
What makes you think burning coins increases on-chain tx volume and confidence...? This is the most crypto thing I've ever heard anyone say. You have to look at this shit logically, not like a memecoin.

**Reply by pcfreak30** [reply]
`Burning coins can boost community confidence, attract more miners, and make the network more secure.` and... This feels like it was written by AI as a marketing pitch.

**Reply by mtomich** [reply]
> `Handshake needs to burn tokens to restore community confidence!` LOL, no. Confidence is increased when things are getting made that actually create adoption (tech, not investors). NgU should not be the primary approach for that.

This 👆🏼

**Comment by TLDSfuture** [comment]
I love handshake
support burn

**Reply by proneelkanth** [reply]
Bro what is name of character on your pfp

**Comment by evbots** [comment]
Support burn. Been saying this for years.

**Comment by lyyMyGitHub** [comment]
Burn it, or the project won't get up. Let's build it together.

**Comment by bishikie** [comment]
Burn Baby Burn 🪙🤝🔥

**Comment by marioo-lab** [comment]
Burn it, and let's move on! 🔥

**Comment by AgaaminT** [comment]
I am mostly in favour of this proposal. Two observations on some of the non-critical suggestions 

1 - Not in favour of tampering with the renewal process. It's fine the way it is 

2 - In favour of on chain records of any kind as long as it does not exceed the 512 bytes allocated to each TLD. 

3 - We can make domain names (Top 1K) and TLDs claimable for a longer period like 10 years.

**Comment by pcfreak30** [comment]
@heytx 

I am stating this here just so its recorded. My current opinions and counter ideas are the following:

- I am open to a burn (though I do not like the idea at all), but ONLY if there is a multisig R&D wallet for the chain for protocol and some ecosystem development (https://github.com/handshake-org/HIPs/discussions/64). Based on what Andrew Lee (purse) said in a prev discord call, @hnsbroker was ok with 10%.
- I am open to adding scarcity on the domain renewals.
- I am open to allowing any kind of record onchain.

That stated, I did some quick research via AI, and we can have a 300 HNS block reward sent to a multi-sig (100 mil lifetime, ~10%). This will be 1,296,000 HNS per month, over ~6.5 years. This is **NOT** an advocacy for just sending 100m coins on a ledger to be dumped, like a faction appears to fear.

This could be increased or reduced based on how fast we want to go vs LP, and community input.

We would also want some sort of means to swap out governance people. There are existing ideas that we can use for that, and I can gather some of that info to share, but it is best we be able to do so, as it would be irresponsible otherwise. Additionally some discussion should probably be made with a document, in a GitHub `.md`, outlining what that group is required to do and what has been agreed on to be funded.

If anyone wants to give **_constructive_** feedback on this, feel free to comment.

**Reply by bishikie** [reply]
Why do you always want to use centralization to plunder everyone's property

**Reply by pcfreak30** [reply]
Give me a way to fund the system that is decentralized (and that isnt go to VC, stay away from us) without any designated group of people making governance decisions and I will happily support that technical plan. Only way I know is a hard fork for every funding proposal.

We need money to get shit done, and no one is funding now, or we wouldn't be talking about it.

I am not a Bitcoiner btw. I do not subscribe to the idea I am asking to steal anything. If people collectively decide to do this and we are doing it for the *public goods R&D of the network*, then we are doing it to prevent problems. While I care about BTC in spirit, there is a lot of BTC worldview ideas I do not subscribe to.

As it stands, I think the burn is worthless, and will either do nothing or will do harm. But if im outvoted on it, I will support it if we can solve the fact HNS has no funding.

And *please* don't suggest to get a job or some other comments. Im grant funded by another chain, and I have not and do not plan on asking HNS for money.

Lastly, decentralization is a means to an end, its never the goal. If absolute property ownership via code and censorship-resistance could be achieved another way, I would support it.

**Reply by AndreiChyrych** [reply]
I hope you do understand that creating a multisig, DAO or any other entity with control of +40% of total supply is not going to solve anything. 

1. We have no idea how much USD we can get for unblocking millions of HNS coins. 
2. If we have no idea about USD value of unlock, you can’t budget. If you can’t budget, you can’t have any plan on what to spend the money on. 
3. If you can’t plan and budget, the whole upside of proposals like reallocation, DAO etc becomes nonexistent. You only get the downside. 

It is a painfully simple concept and I have no idea why it is so hard to understand. Please try to understand what I’m talking about before posting a big comment. 

P.S According to your logic every business founder, owner should get a grant from someone to start a business. And if you don’t have any grants, you don’t start a business.

**Reply by Falci** [reply]
> Give me a way to fund the system that is decentralized

Each company pays for devs to do what each of them decides is the best, for them or for the community, like Namebase, Kyokan, Purse.io... and others did.


> We need money to get shit done

True. Agree.

>and no one is funding now

Namebase is. Maybe nobody is funding you/your ideas.


> If people collectively decide to do this

Thing is: people don't agree on this.

> I will support it if we can solve the fact HNS has no funding

AFAIK, burning proposal is not trying to solve funding issues.

> Im grant funded by another chain, and I have not and do not plan on asking HNS for money.

Would you consider sponsoring some developers then?

**Reply by pcfreak30** [reply]
> I hope you do understand that creating a multisig, DAO or any other entity with control of +40% of total supply is not going to solve anything.
> 
> 1. We have no idea how much USD we can get for unblocking millions of HNS coins.
> 2. If we have no idea about USD value of unlock, you can’t budget. If you can’t budget, you can’t have any plan on what to spend the money on.
> 3. If you can’t plan and budget, the whole upside of proposals like reallocation, DAO etc becomes nonexistent. You only get the downside.
> 
> It is a painfully simple concept and I have no idea why it is so hard to understand. Please try to understand what I’m talking about before posting a big comment.
> 
> P.S According to your logic every business founder, owner should get a grant from someone to start a business. And if you don’t have any grants, you don’t start a business.

`We have no idea how much USD we can get for unblocking millions of HNS coins.` True... But right now we have 0, so for me, im trying to create progress at all.

`If we have no idea about USD value of unlock, you can’t budget. If you can’t budget, you can’t have any plan on what to spend the money on.` If we know what we need, we can figure out estimations, and then eventually be able to fund what is needed. I view it in the opposite approach planning wise vs you.

` According to your logic every business founder, owner should get a grant from someone to start a business.`? no. im not stating any of that at all. My view is only public goods which means the core chain and ecosystem projects which are FOSS need funding. I am hard-lined against funding with the intent of being capital to a business.

Lastly my comment was suggesting 5-10% of whats unclaimed, not 40%+.

**Reply by pcfreak30** [reply]
> > Give me a way to fund the system that is decentralized
> 
> Each company pays for devs to do what each of them decides is the best, for them or for the community, like Namebase, Kyokan, Purse.io... and others did.
> 
> > We need money to get shit done
> 
> True. Agree.
> 
> > and no one is funding now
> 
> Namebase is. Maybe nobody is funding you/your ideas.
> 
> > If people collectively decide to do this
> 
> Thing is: people don't agree on this.
> 
> > I will support it if we can solve the fact HNS has no funding
> 
> AFAIK, burning proposal is not trying to solve funding issues.
> 
> > Im grant funded by another chain, and I have not and do not plan on asking HNS for money.
> 
> Would you consider sponsoring some developers then?

`Namebase is. Maybe nobody is funding you/your ideas.`. Namebase, as I see it, is funding themselves, not community projects or core development outside their obligations and interests to the chain. So I don't view that as funding the chain, I see it as them building a business *on* the chain.

`Thing is: people don't agree on this.` tbh its been hard to know what exactly they do or don't agree on with a lot of discussion that's overlapped. The point is im posting the ideas.

`AFAIK, burning proposal is not trying to solve funding issues.` I know, and due to that, I have no support for a burn proposal alone and will only support it if we fix funding in the same upgrade. Otherwise, I would say do nothing.

`Would you consider sponsoring some developers then?`

I said im grant funded, I didn't say im rich. Im funded for myself and for hiring for some Lume project work. I don't have 500k+ sitting in a bank account.

**Reply by AndreiChyrych** [reply]
> Lastly my comment was suggesting 5-10% of whats unclaimed, not 40%+.

Every time someone brings up reallocation ideas, it quickly goes towards 40% 🤷🏻‍♂️ That’s why I’m referring to the whole amount of unclaimed coins.

**Reply by Falci** [reply]
> Namebase, as I see it, is funding themselves, not community projects or core development outside their obligations and interests to the chain. So I don't view that as funding the chain, I see it as them building a business _on_ the chain.

This is the decentralisation you asked about. If you believe more efforts should be dedicated somewhere else, you are free to sponsor (or convince someone else to sponsor) a team (or you) to work on whatever. This way we don't need a centralised DAO.

Maybe that is all we need: more companies "building businesses on the chain"

**Reply by pcfreak30** [reply]
` If you believe more efforts should be dedicated somewhere else, you are free to sponsor (or convince someone else to sponsor) a team (or you) to work on whatever. This way we don't need a centralised DAO.`

And... the fact that's not happening, and especially no one is donating *to* a community fund is the crux issue (2-3k isn't much (per discord). And I would be funding things or doing it myself if I was able to, but I both am time committed right now and am frankly broke. Me contributing more to HNS is longer term for me, im doing what I am able to for now.

So, I see centralization for bootstrapping a chain as an acceptable short to mid term trade off to ensure the chain can self-sustain itself, if ppl are not pitching in. From all the chatter I have seen, that is not the Bitcoin way, but I am also not a Bitcoiner :upside_down_face: .

**Reply by pcfreak30** [reply]
> > Lastly my comment was suggesting 5-10% of whats unclaimed, not 40%+.
> 
> Every time someone brings up reallocation ideas, it quickly goes towards 40% 🤷🏻‍♂️ That’s why I’m referring to the whole amount of unclaimed coins.

And I mentioned the fact that hnsbroker, a stated maxi in position, was ok with 10% apparently after andrew explained why. So, im trying to compromise because I care most about the funding issue even if I see the burn as an utter waste of time and money (literally).

**Reply by AndreiChyrych** [reply]
> So, I see centralization for bootstrapping a chain as an acceptable short to mid term trade off to ensure the chain can self-sustain itself

But HNS works as intended so the chain is in the self sustained state already, no? 

There’s no secure and decentralized alternative to the internet root zone apart from HNS. All the other crypto projects are doing something else and decentralization is not their main focus to say the least. 

Handshake is extended to several other blockchains, L1s and L2s. That’s remarkable achievement on its own. 

We lack use cases, marketing, biz dev activity and excitement about the technology itself but in my opinion this is something that only businesses using HNS will solve.

**Reply by pcfreak30** [reply]
> But HNS works as intended so the chain is in the self sustained state already, no?

No, I dont see the chain as treading water and no progress (or very little) on adoption or or R&D as self-sustaining.

There are development the chain needs, both as core, and eco public goods which will fix a number of problems and enable utility and more adoption. All of what I want to see are just public goods development, and nothing thats direct investment to any business using the chain.

The idea of businesses doing everything is fine, and if they contribute to a fund, I will thank them. But its not happening, and hasn't happened. So based on that, I have to advocate for solutions.

**Comment by TobesVibration** [comment]
I support this proposal. It’s really interesting. Slightly concerned about the top 10,000 being squatted on but apart from that everything seems really good. I definitely think burning the extra tokens is the way to go. Lots of utility for Handshake coming on Cardano soon thanks to Digitalis and Blink Labs 🧪

**Comment by ghost** [comment]
I'm happy to help fund core developers. Thank you for your contributions to the protocol 🤝

![burn](https://github.com/user-attachments/assets/6f8bfc10-136a-4e2c-a499-8cf7fab7fc17)

**Comment by skyinclude** [comment]
Our stance On this proposal.

For burn - there is too much pushback on a reallocation. Core devs such as Nodar and Zipkin are opposed to any reallocation and imagine miners would be too. The tech and the infrastructure are built first for the developers and maintained by the miners - we are the users and need to respect the foundation of tech and upkeep we are built on.

For Alexa 1k + ICANN TLD for claim again, releasing Alexa 1001-10k for auction. But against too much HNS coin allocated to that Alexa 1k + ICANN TLD, they had 4+years to claim that coin, give them enough now to renew for 100+ years.

For “ReRecord” and having DNS records on the TLD root / on-chain. This will add a lot of direct utility for names and give more transactions. Those who want to still point to name servers and do off-chain or elsewhere can still do so. Happy our https://rerecord.skyinclude.com was included here.

For fee renewal and reducing block size from 600 to 200 - have we done math on this? I’d just want to be careful that WHEN (not IF) we get mass adoption, we become another “expensive Ethereum L1” and people are complaining about high fees. Or is this “relatively easy to adjust” later? For now it is acceptable, but if we are a global root zone this may become costly for those in international markets.

**Comment by HNSbroker** [comment]
Burning the airdrop, making ICANN TLDs claimable, and adding records on-chain is the way forward. Is there an effective way to burn the coins, perhaps through an auction-style burn? This would prevent us from facing the same situation again.

Fee and renewal limit adjustments should be reconsidered; miners will still earn enough. Transactions will peak with on-chain records. Jesse's idea for a dynamic fee makes sense as long as it doesn't negatively impact users with 3-5 records.

I also support Sajan's suggestion of opening the Alexa 1000-10000 names.

Thanks, Nole, for making this proposal—it's clearly the direction the community wants to go. Let's set the plan in motion as soon as possible.

**Comment by mtomich** [comment]
Burning is idiotic. It's a no from me.

**Comment by AndreiChyrych** [comment]
>If there is community consensus, including support from Namebase, @nodech has agreed to write the code and be the lead developer. There should also be additional developers to review and assist where needed.

What is the definition of community consensus? If there’s no consensus about what to do with the coins, maybe we should just focus on adding records and unblocking domains?

**Reply by mtomich** [reply]
I don't see a consensus. Logical folks realize a burn adds no long term value. Value is ONLY created with a use case for TLDs. When TLDs have a use case/purpose $HNS will have value as well.

**Reply by AndreiChyrych** [reply]
Classic PoW chain: 

If you want to own a coin you either mine or buy from miners. There’s no way around it so every dollar invested increases the value of the chain: 1) You buy equipment 2) You exchange $ for HNS. Real capital goes in, real capital goes out (in case anyone sells HNS for USD, EUR etc). 

Fuckery PoW: 

Classic model but you can also get coins if you talk good and promise a lot. You don’t necessarily need to provide any real capital: equipment or $. You just have to convince multisig signers to transfer you the check. Why should any miner take such risk? You provide capital, work hours etc but a random 20 years old biz dev/marketing twitter guru can fck it all up for you? That’s a lot of risk. 

If there’s no miners, there’s no blocks. No blocks = no blockchain. Since HNS is PoW and not PoS, mining matters. Miners provide security. No security = no base value of any TLD. TLDs on HNS are only valuable because of mining. Everything else comes on top. 

While I agree that burning might not provide any significant price impact, burning unclaimed coins will secure HNSs legitimacy among miners (since there will be only 2 ways of getting coins). 

Here’s why legitimacy matters. You want to list HNS on exchanges? Mining pools are typically the most interested, liquid and “ready for action” units. They own shtloads of coins, they can cover the fees, provide MM with liquidity etc. Your regular TLD holder can’t even be compared to decent mining pool and their arsenal of tools. 
 
Listing on exchanges will allow greater group of people to speculate on names and coins. Speculation on names and coins with drive the price of HNS up. Price goes up = everybody happy. 

Ethically sound foundation will attract decent business partners. Minimal counter party risk + great use case will keep them for years.
